src/main/java/org/sonarlint/intellij/config/global/SonarLintGlobalSettingsPanel.java

```diff
@@ -20,14 +20,10 @@
 package org.sonarlint.intellij.config.global;
 
 import com.intellij.openapi.ui.VerticalFlowLayout;
-
-import javax.swing.BorderFactory;
-import javax.swing.JCheckBox;
-import javax.swing.JComponent;
-import javax.swing.JPanel;
-import java.awt.BorderLayout;
+import java.awt.*;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Explicitly import the specific classes needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2208)